src/rest-server/deploy/legacy_user_migrate.py

```diff
@@ -138,6 +138,60 @@ def secret_data_prepare_v2_group(self, groupname):
         post_data_dict['data'] = group_dict
         return post_data_dict
 
+    def load_v2_groups(self):
+        ns_pai_group_list = self.list_all_secrets_from_namespace(self.secret_ns_group_v2)
+        decode_group_list = []
+        vc_set = set()
+        for group in ns_pai_group_list:
+            meta_dict = dict()
+            meta_dict['name'] = bytes.fromhex(group.metadata.name).decode('utf-8')
+            group_dict = {
+                'groupname': str(base64.b64decode(group.data['groupname'].encode('utf-8')), 'utf-8'),
```

Comment:
Are there any update or change happening at groupname, description or externalName? If not, you can use the value from k8s directly. And I think decode is unnecessary.

Reply:
See above